spring-cloud-gcp-kotlin-samples/pom.xml

```diff
@@ -16,6 +16,19 @@
 		<module>spring-cloud-gcp-kotlin-app-sample</module>
 	</modules>
 
+	<!-- The Spring Cloud GCP BOM will manage spring-cloud-gcp version numbers for you. -->
```

Comment:
10 lines up, do you want to add the same comment about
```xml
 <!-- Your own application should inherit from spring-boot-starter-parent -->
```
?

Reply:
Done.